lib/harvest_utils.rb

```diff
@@ -437,11 +438,16 @@ def self.normalize_rights(doc, string_to_search)
       node_update = doc.search(string_to_search, "dc" => "http://purl.org/dc/elements/1.1/")
       node_update.each do |node_value|
         if node_value.inner_html.downcase.starts_with?("http")
-          node_value.inner_html = node_value.inner_html.downcase
+          conform_url node_value
```

Comment:
Hmm. Actually, looking at it, the patch no longer updates  `node_value.inner_html`